appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -726,7 +726,9 @@ private YaVersion() {
 
   // For SLIDER_COMPONENT_VERSION 1:
   // - Initial version.
-  public static final int SLIDER_COMPONENT_VERSION = 1;
+  // For SLIDER_COMPONENT_VERSION 2:
+  // - Added the property to allow for the removal of the Thumb Slider
+  public static final int SLIDER_COMPONENT_VERSION = 2;
```

Comment:
You also need to update YOUNG_ANDROID_VERSION. Search for it in the code, what you need to do is obvious (or should be :-) )